addons/event/views/event_views.xml

```diff
@@ -698,6 +698,7 @@
                         domain="[('my_activity_date_deadline', '=', context_today().strftime('%Y-%m-%d'))]"/>
                     <filter invisible="1" string="Future Activities" name="activities_upcoming_all"
                         domain="[('my_activity_date_deadline', '&gt;', context_today().strftime('%Y-%m-%d'))]"/>
+                    <filter string="Last 30 days" name="filter_last_month_activities" domain="[('create_date','&gt;', (context_today() - datetime.timedelta(days=30)).strftime('%Y-%m-%d'))]"/>
```

Comment:
filter_last_month_creation maybe ? As it is not linked to write_date, just the creation date ...